pages/_app.js

```diff
@@ -1,15 +1,31 @@
 import Layout from "../components/Layout";
 import GlobalStyles from "../styles/GlobalStyles";
 
+import {useRouter} from "next/router";
+
 function MyApp({Component, pageProps}) {
-  return (
-    <>
-      <GlobalStyles />
-      <Layout>
+  const router = useRouter();
+  const {asPath} = router;
```

Comment:
Maybe you can write this like: 
```suggestion
  const {asPath}  = useRouter();
```

Reply:
done!